features/step_definitions/where_did_you_hear_about_us_report_steps.rb

```diff
@@ -0,0 +1,28 @@
+Given(/^there are existing where did you hear about us records$/) do
+  create_list(:where_did_you_hear, Kaminari.config.default_per_page + 1)
```

Comment:
Personal preference so happy to go withy the majority, but I prefer not to use the ```FactoryGirl``` magic and instead make it expicit.
```
FactoryGirl.create_list(...)
```